api/include/opentelemetry/common/key_value_iterable.h

```diff
@@ -19,16 +19,31 @@ class KeyValueIterable
    * Iterate over key-value pairs
    * @param callback a callback to invoke for each key-value. If the callback returns false,
    * the iteration is aborted.
-   * @return true if every key-value pair was iterated over
+   * @return true if every key-value pair was iterated ove
    */
-  virtual bool ForEachKeyValue(
-      nostd::function_ref<bool(nostd::string_view, common::AttributeValue)> callback) const
-      noexcept = 0;
+  virtual bool ForEachKeyValue(nostd::function_ref<bool(nostd::string_view, common::AttributeValue)>
```

Comment:
Is there a reason this is re-formatted? You should be able to avoid this change.

Reply:
I used the `./tools/format.sh` with the latest version (clang-10) and that caused this formatting -- that's the reason the CI for formatter is still failing. I've fixed this one. I'll see what could be done to fix the formatting issues at other places.

Reply:
I got access to a Ubuntu machine, and have fixed the formatter at all places -- the formatter for CI is passing now